tests/test_float16.py

```diff
@@ -87,6 +89,21 @@ def range_body(iter_n, cond, total):
             self.assertTrue(np.allclose(expected, actual2))
             self.assertTrue(actual2.dtype == np.float32)
 
+    def test_convert_to_float16(self):
+        # load fp32 model from data
+        model32_name = "image_classifier32.onnx"
```

Comment:
I don't see you upload this file into repo. Do we miss it?

In addition, is it possible to construct a new model here instead of uploading an existing ONNX file for test? It will help others to understand what function is going to cause this issue and update it easily in future, if necessary.

Reply:
in //tests/data/image_classifier32.onnx